.github/workflows/module-lint.yml

```diff
@@ -0,0 +1,69 @@
+name: Module Lint
+
+# runs at minute 10 of the 1st hour of every Sunday
+on:
+  schedule:
+    - cron: '10 1 * * 0'
+
+jobs:
+  prepare:
+    name: Prepare
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
+          cache: 'yarn'
+      - name: Install Yarn dependencies
+        run: yarn --immutable
+
+  module-lint:
+    name: Module Lint
+    runs-on: ubuntu-latest
+    needs:
+      - prepare
+    strategy:
+      matrix:
+        node-version: [16.x, 18.x, 20.x]
+        projects:
+          [
+            { name: 'auto-changelog', team: 'S051YKC9350' },
+            { name: 'create-release-branch', team: 'S051YKC9350' },
+            { name: 'module-lint', team: 'S051YKC9350' },
+            { name: 'utils', team: 'S051YKC9350' },
+          ]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'yarn'
+      - run: yarn --immutable --immutable-cache
```

Comment:
Do we need to run `module-lint` for multiple Node version? We do this for tests to ensure that the library/tool in question works for multiple Node versions, since there may be incompatibility issues, but for `module-lint`, I feel like the results should be the same across Node versions. So we should be able to use the Node version specified in `.nvmrc`:

``` suggestion
  module-lint:
    name: Module Lint
    runs-on: ubuntu-latest
    strategy:
      matrix:
        projects:
          [
            { name: 'auto-changelog', team: 'S051YKC9350' },
            { name: 'create-release-branch', team: 'S051YKC9350' },
            { name: 'module-lint', team: 'S051YKC9350' },
            { name: 'utils', team: 'S051YKC9350' },
          ]
    steps:
      - uses: actions/checkout@v3
      - uses: actions/setup-node@v3
        with:
          node-version-file: '.nvmrc'
          cache: 'yarn'
      - run: yarn --immutable